pkgs/development/python-modules/yamllint/default.nix

```diff
@@ -45,7 +52,7 @@ buildPythonPackage rec {
     description = "A linter for YAML files";
     homepage = "https://github.com/adrienverge/yamllint";
     changelog = "https://github.com/adrienverge/yamllint/blob/v${version}/CHANGELOG.rst";
-    license = licenses.gpl3Plus;
+    license = licenses.gpl3Only;
```

Comment:
that doesn't look right - most files seems to be licensed gpl3+ according to the file headers, e.g. https://github.com/adrienverge/yamllint/blob/b05e028c5881819161d11cb543fd96a30c06cceb/yamllint/cli.py#L6

Reply:
pyproject.yaml says the opposite: https://github.com/adrienverge/yamllint/blob/b05e028c5881819161d11cb543fd96a30c06cceb/pyproject.toml#L6, so one of then has to be fixed.

Reply:
Opened PR upstream: https://github.com/adrienverge/yamllint/pull/580